main.go

```diff
@@ -130,9 +136,21 @@ func main() {
 	}).SetupWithManager(mgr); err != nil {
 		klog.Exitf("unable to create PerformanceProfile controller : %v", err)
 	}
+
+	// configure webhook server
+	webHookServer := mgr.GetWebhookServer()
+	webHookServer.Port = webhookPort
+	webHookServer.CertDir = webhookCertDir
+	webHookServer.CertName = webhookCertName
+	webHookServer.KeyName = webhookKeyName
+
 	if err = (&performancev1.PerformanceProfile{}).SetupWebhookWithManager(mgr); err != nil {
-		klog.Exitf("unable to create PerformanceProfile webhook : %v", err)
+		klog.Exitf("unable to create PerformanceProfile v1 webhook : %v", err)
```

Comment:
What about v1alpha1?

Reply:
it should be registered only for hub

Reply:
Aha, so  why do we need the one below? performancev2 ?

Reply:
hm, I need to refresh my memory, so what I talked about before is wrong(does not relate to hub)
so the validation webhook implemented only for v2, so we call it for v1 to register conversion webhook and for v2 to register validating webhook

Reply:
got it, thanks